### PR TITLE
added documentation files 

### PR DESCRIPTION
**Pull request by MarcinGuriDrozdz**

### CODE REVIEW

**Review by manskimarcin — COMMENTED** [review]
Wszystko gra

**Review by manskimarcin — APPROVED** [review]

File: Dokumentacja/deprecated-list.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+<li><a href="index.html?deprecated-list.html" target="_top">Frames</a></li>
+<li><a href="deprecated-list.html" target="_top">No&nbsp;Frames</a></li>
+</ul>
+<ul class="navList" id="allclasses_navbar_bottom">
+<li><a href="allclasses-noframe.html">All&nbsp;Classes</a></li>
+</ul>
+<div>
+<script type="text/javascript"><!--
+  allClassesLink = document.getElementById("allclasses_navbar_bottom");
+  if(window==top) {
```
Comment (BartoszMyrcha) [inline Dokumentacja/deprecated-list.html]: What is a purpose of this part of code?

**Review by BartoszMyrcha — CHANGES_REQUESTED** [review]

File: Dokumentacja/deprecated-list.html (hunk 1 above)
Comment (MarcinGuriDrozdz) [reply Dokumentacja/deprecated-list.html]: Ask google

Comment (BartoszMyrcha) [reply Dokumentacja/deprecated-list.html]: kk

**Review by BartoszMyrcha — APPROVED** [review]

**Pull request merged**